Dockerfiles/server_config.cfg

```diff
@@ -0,0 +1,9 @@
+[Webserver]
```

Comment:
Why is this file duplicated now? If I interpret correctly, you copy everything in the Planarally folder, including the cfg file there, then copy this one over the original one. Am I missing something?

Reply:
I'm only copying the content of the folder PlanarAlly/, not the entire thing.